src/scripts/main.js

```diff
@@ -1,7 +1,22 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement(`div`);
+
+  div.classList.add(`notification`, type);
+
+  div.innerHTML = `
+  <h2 class = "title">${title}</h2>
+  <p>${description}</p>`;
```

Comment:
якщо не хочеш користуватися append тоді форматуй нормально код в лапках
```suggestion
  div.innerHTML = `
    <h2 class="title">${title}</h2>
    <p>${description}</p>
  `;
```